apollo-runtime/src/main/java/com/apollographql/apollo/internal/subscription/RealSubscriptionManager.kt

```diff
@@ -299,10 +299,7 @@ class RealSubscriptionManager(private val customScalarAdapters: CustomScalarAdap
       val normalizer = responseNormalizer.invoke()
       val subscription = subscriptionRecord!!.subscription
       val response = try {
-        MapResponseParser.parse(message.payload, subscription, customScalarAdapters).let {
-          val records = it.data?.let { subscription.normalize(it, customScalarAdapters, normalizer as ResponseNormalizer<Map<String, Any>?>) }
-          it.copy(dependentKeys = records.dependentKeys())
-        }
+        MapResponseParser.parse(message.payload, subscription, customScalarAdapters)
```

Comment:
I think this breaks storing subscriptions results in the cache since `normalizer.records()` (used in notifyOnResponse below) was computed as a side effect of this `normalize()` call.

I wonder if we have tests for that. We should but for some reason, Github Actions does not trigger here...

Reply:
Ah good call let me re-introduce it there.